compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -197,13 +197,30 @@ namespace jank::runtime
       fn->unique_name = fn->name;
       codegen::llvm_processor cg_prc{ wrapped_exprs, module, codegen::compilation_target::module };
       cg_prc.gen().expect_ok();
-      write_module(std::move(cg_prc.ctx)).expect_ok();
+      write_module(cg_prc.ctx->module_name, cg_prc.ctx->module).expect_ok();
     }
 
     assert(ret);
     return ret;
   }
 
+  void context::eval_cpp_string(native_persistent_string_view const &code) const
+  {
+    profile::timer const timer{ "rt eval_cpp_string" };
+    auto &partial_tu{ jit_prc.interpreter->Parse({ code.data(), code.size() }).get() };
+
+    /* Writing the module before executing it because `llvm::Interpreter#Execute`
+     * moves the `llvm::Module` held in the `PartialTranslationUnit`. */
+    if(truthy(compile_files_var->deref()))
+    {
+      auto module_name{ runtime::to_string(current_module_var->deref()) };
+      write_module(module_name, partial_tu.TheModule).expect_ok();
+    }
+
+    auto err(jit_prc.interpreter->Execute(partial_tu));
+    llvm::logAllUnhandledErrors(std::move(err), llvm::errs(), "error: ");
```

Comment:
So here, we just swallow the error and write to stderr. I think it's important to return this error, rather than just make noise about it. If we fail to parse, this will throw, but if we fail to execute, this should behave the same. Either they both return errors or they both throw.